focus-autosave-mode.el

```diff
@@ -73,12 +75,12 @@
 (defun focus-autosave-save-marked ()
   "Save the marked buffers and remove the killed ones from the list."
   (setq focus-autosave-buffer-list
-        (delete-if-not #'buffer-live-p
-                       focus-autosave-buffer-list))
-  (mapcar (lambda (buffer)
-            (with-current-buffer buffer
-              (save-buffer)))
-          focus-autosave-buffer-list)
+        (cl-delete-if-not 'buffer-live-p
+                          focus-autosave-buffer-list))
+  (mapc (lambda (buffer)
```

Comment:
`mapc` should be used when mapping for side effects only, because it doesn't build a redundant result list.